File: src/PizzaPlus/Toppings.java
package PizzaPlus;

import java.io.Serializable;

public class Toppings implements Serializable {

    private String toppings;
    private String type;
    private double priceforSmall;
    private double priceforMedium;
    private double priceforLarge;
    private int quantityOunces;
    private double toppingPrice;

    public Toppings(String toppings, String type, double priceforSmall, double priceforMedium, double priceforLarge){
        this.toppings = toppings;
        this.type = type;
        this.priceforSmall = priceforSmall;
        this.priceforMedium = priceforMedium;
        this.priceforLarge = priceforLarge;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public double getPriceforSmall() {
        return priceforSmall;
    }

    public void setPriceforSmall(double priceforSmall) {
        this.priceforSmall = priceforSmall;
    }

    public double getPriceforMedium() {
        return priceforMedium;
    }

    public void setPriceforMedium(double priceforMedium) {
        this.priceforMedium = priceforMedium;
    }

    public double getPriceforLarge() {
        return priceforLarge;
    }

    public void setPriceforLarge(double priceforLarge) {
        this.priceforLarge = priceforLarge;
    }

    public double getToppingPrice() {
        return toppingPrice;
    }

    public void setToppingPrice(double toppingPrice) {
        this.toppingPrice = toppingPrice;
    }

    public String getToppings() {
        return toppings;
    }
    public int getQuantityOunces() {
        return quantityOunces;
    }
    public void setToppings(String toppings) {
        this.toppings = toppings;
    }
    public void setQuantityOunces(int quantityOunces) {
        this.quantityOunces = quantityOunces;
    }
}

File: src/PizzaPlus/Toppings.kt
package PizzaPlus

import PizzaPlus.Customer.newCustomerMenu
import PizzaPlus.Customer.displayCustomerInfo
import PizzaPlus.Customer.add
import PizzaPlus.Customer.firstName
import PizzaPlus.Customer.lastName
import kotlin.jvm.JvmStatic
import PizzaPlus.Customer
import PizzaPlus.Order
import PizzaPlus.Pizza
import PizzaPlus.Toppings
import java.io.ObjectOutputStream
import java.io.FileOutputStream
import java.io.FileNotFoundException
import java.io.IOException
import java.io.ObjectInputStream
import java.io.FileInputStream
import java.lang.ClassNotFoundException
import kotlin.Throws
import PizzaPlus.XmlHandler
import java.io.File
import java.io.Serializable
import javax.xml.parsers.SAXParserFactory

class Toppings(
    var toppings: String,
    var type: String,
    var priceforSmall: Double,
    var priceforMedium: Double,
    var priceforLarge: Double
) : Serializable {
    var quantityOunces = 0
    var toppingPrice = 0.0

}